haskell/test/Stablecoin.hs

```diff
@@ -0,0 +1,107 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+module Stablecoin
+  ( spec_Stablecoin
+  , spec_StablecoinNettest
+  , stablecoinNettestReal
+  ) where
+
+import Test.Hspec
+
+import Tezos.Core
+import Data.Maybe
```

Comment:
It should be in another group and be explicit (`Test.HSpec` as well).

Reply:
Have fixed.